### ISSUE TITLE
Bootstrapping CLI accounts is not working

### ISSUE THREAD

**Issue by geigerzaehler** [radicle-dev/radicle-registry#314]
When using the CLI for the first and no account exists yet it is impossible to create an account. The following command fails

```
$ radicle-registry-cli --tx-author default account generate default
error: Invalid value for '--tx-author <account_name>': Could not find local account named 'default'
```

**Comment by NunoAlexandre** [comment]
@geigerzaehler that command doesn't quite make sense. You are unnecessarily setting the `--tx author` to an account that doesn't exist.

You should just do as follows, where `default` is the name you are giving to the account being generated:

``` bash
radicle-registry-cli account generate default
```

**Comment by geigerzaehler** [comment]
> ```shell
> radicle-registry-cli account generate default
> ```

This does not work because `--tx-author` is a required option.

**Comment by NunoAlexandre** [comment]
Ah, right! That's an issue introduced at #305.

Good thing I am already raising a PR to fix #283 where this is solved too.
